controller/workitem.go

```diff
@@ -433,7 +433,7 @@ func ConvertJSONAPIToWorkItem(ctx context.Context, appl application.Application,
 			if err != nil {
 				return errors.NewBadParameterError("data.relationships.iteration.data.id", *d.ID)
 			}
-			if _, err = appl.Iterations().Load(ctx, iterationUUID); err != nil {
+			if _, err := appl.Iterations().Exists(ctx, iterationUUID.String()); err != nil {
```

Comment:
@kwk that is why I asked here https://github.com/almighty/almighty-core/pull/1415#discussion_r122726636 :)